data/cybox/nozomi/nozomi_alerts_22022024.json

```diff
@@ -0,0 +1,563 @@
+                    "severity": 80,
+                    "time_observed": "2023-12-12T07:10:57.012Z",
+                    "src_ip_ref": "1",
+                    "x_alert_status": "open",
+                    "name": "Sigma rule match",
+                    "x_cause": "Rule-dependent. A suspicious local event has been detected on a machine.",
+                    "ioc_refs": [
+                        "10"
+                    ],
+                    "x_solution": "Rule-dependent. Verify the device configuration and status, and the possible presence of malicious processes.",
+                    "description": "The following suspicious local event was detected on this host: \"Detects the execution of renamed PuTTY Plink to perform data exfiltration through tunnel port forwarding\".",
+                    "x_is_incident_alert": false,
+                    "x_is_cybersecurity_alert": true,
+                    "finding_type": "alert",
+                    "start": "2023-12-12T07:10:57.012Z",
+                    "rule_names": [
+                        "sigma_rules"
+                    ],
+                    "x_sensor_host": "EC2AMAZ-XXXXX",
+                    "x_alert_type_id": "SIGN:SIGMA-RULE",
+                    "x_rule_id": "5f9b24dd-fa05-401c-ad21-240b3f761c9f"
+                },
+                "1": {
+                    "type": "ipv4-addr",
+                    "value": "1.1.1.1",
+                    "x_nozomi_info_ref": "3"
+                },
+                "2": {
+                    "type": "network-traffic",
+                    "src_ref": "1",
+                    "protocols": [
+                        "unknown"
+                    ]
+                },
+                "3": {
+                    "type": "x-nozomi-info",
+                    "label": "EC2AMAZ-XXXXX",
+                    "is_public": false
+                },
+                "4": {
+                    "type": "process",
+                    "pid": 1512,
+                    "creator_user_ref": "5",
+                    "parent_ref": "8",
+                    "binary_ref": "10",
+                    "command_line": "pip  install -r requirements-dev.txt"
+                },
+                "5": {
+                    "type": "user-account",
+                    "user_id": "Administrator@EC2AMAZ-XXXXX"
+                },
+                "6": {
+                    "type": "directory",
+                    "path": "C:\\Windows\\System32"
+                },
+                "7": {
+                    "type": "file",
+                    "name": "cmd.exe",
+                    "parent_directory_ref": "6"
+                },
+                "8": {
+                    "type": "process",
+                    "command_line": "C:\\Windows\\System32\\cmd.exe",
+                    "binary_ref": "7"
+                },
+                "9": {
+                    "type": "directory",
+                    "path": "C:\\UDI automation\\venv\\Scripts"
```

Comment:
I'm not sure if a path can be considered PI, but this looks unique to the environment. Can you replace this path with an obscured one?